### PR TITLE
Camera calibration code

### PR DESCRIPTION
**Pull request by AG6GR**

### CODE REVIEW

File: capture_image.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from operator import add
+
+
+# finds rotation and translation of vision targets
+class CaptureImage:
+
+    # initilaze variables
+    def __init__(self, detector):
+
+        self.detector = detector
```
Comment (AG6GR) [inline capture_image.py]: If we change this constructor to just take a generator, then we can pass in a VideoLiveGenerator without having to import all the detector code.

File: run_capture.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    angle = input("angle: ")
+    distance = input("distance: ")
+
+    while True:
+        c.update_frame()
+        key = cv2.waitKey(wait_time)
+        # pos = (generator.get_frame().shape[1] /2, generator.get_frame().shape[0]/2)
+        pos = (100, 50)
+
+        cv2.drawMarker(generator.get_frame(),pos, (255, 0, 0), cv2.MARKER_CROSS, 20, 10, cv2.LINE_8)
```
Comment (AG6GR) [inline run_capture.py]: We're already grabbing a frame inside CaptureImage.update_frame(). The marker drawing code should probably go in there.

File: run_capture.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import time
+import argparse
+import cv2
+import os
+from capture_image import CaptureImage
+from video_live_generator import VideoLiveGenerator
+from power_port_detector import PowerPortDetector
```
Comment (AG6GR) [inline run_capture.py]: Detector should not be needed. The corresponding detector files should be removed as well.

File: video_live_generator.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class VideoLiveGenerator:
+    def __init__(self, input_port):
+        self.input_port = input_port
+        self.input = cv2.VideoCapture(int(input_port))
+        self.set_camera_settings(str(input_port))
+
+        self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 640)
+        self.input.set(cv2.CAP_PROP_FRAME_HEIGHT, 480)
```
Comment (AG6GR) [inline video_live_generator.py]: Our webcam supports up to 1080p resolution. Can we set the resolution higher?

**Comment by AG6GR** [comment]
See code comments above.

File: capture_image.py [hunk 5]
```diff
@@ -12,18 +12,30 @@
 # finds rotation and translation of vision targets
 class CaptureImage:
 
-    # initilaze variables
-    def __init__(self, detector):
+    # # initilaze variables
+    # def __init__(self, detector):
```
Comment (AG6GR) [inline capture_image.py]: Do you still need this code? If not, delete it. You can see the history in Git, no need to keep old code commented.

File: video_live_generator.py [hunk 6]
```diff
@@ -9,8 +9,8 @@ def __init__(self, input_port):
         self.input = cv2.VideoCapture(int(input_port))
         self.set_camera_settings(str(input_port))
 
-        self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 640)
-        self.input.set(cv2.CAP_PROP_FRAME_HEIGHT, 480)
+        self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 1080)
```
Comment (AG6GR) [inline video_live_generator.py]: 1080p refers to the number of vertical lines, see https://en.wikipedia.org/wiki/1080p

**Review by AG6GR — COMMENTED** [review]
Almost there. I see you have a few style errors, please check the style check action output on the pull request.

**Review by shivanisahni — APPROVED** [review]

**Pull request merged**